### DISCUSSION TITLE
Welcome to Tampermonkey Discussions!

### DISCUSSION THREAD

**Discussion by derjanb** [Tampermonkey/tampermonkey#1094] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by coolcomfort** [comment]
Do you know how I can delete all GM_values? I have a bunch of GM_values plugged in but I don't want to individually delete each one.
